### ISSUE TITLE
Keyboard does not show when trying to entering PIN after wallet timeout on the background

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#1418]
### Description of problem
After leaving the app open in the background for some time, when the user reactivates the app, and it shows the Enter PIN screen, the keyboard does not pop up when taping the PIN field

### Environment
- [ ] Occurs on Android - Have not observed or received any report for Android
- [x] Occurs on iOS - Someone contacted our support email, and CSB has reported a few similar occasions

Build #:
- 1.0.10 Build (1097)

Android Device Model: 

iOS Device Model: 
- iOS 16.6
- Model name: iPhone12
- Model Number: MGJ33VC/A

### Severity
- [ ] High
- [ ] Medium
- [x] Low

**Comment by bryce-mcmath** [comment]
Hard to troubleshoot because I can't reproduce. Some things to suggest to users who experience this issue: turn off bluetooth, disable Prefer Cross-Fade Transitions, disable Reduce Motion (in iPhone settings)

I might try adding a `Keyboard.dismiss()` on screen focus to see if that eliminates the issue, just have to make sure it doesn't cause any problems

**Comment by jleach** [comment]
I think I've seen similar behaviour on occasion but I cannot reproduce this. I tried reduced the timeout to 1 min, then tried letting it timeout in various ways: 1) while it the foreground, let the phone go to sleep, 2) background the app, then let the phone go to sleep, and 3) background the app and keep the phone active. Repeated testing 10x. All worked as expected.

The app does open slightly different depending on if its opened from the the "home" screen or "search" screen. The search screen tends to not automatically show the keyboard but does display it once the field it touched.

Fixed by the update to RN 0.72 perhaps? That was merged on Sept 28th.
